Contributing.md

```diff
@@ -4,14 +4,14 @@
 [![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
 
 - Take a look at the existing [Issues](https://github.com/EddieHubCommunity/LinkFree/issues) or [create a new issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose)!
-- [Fork the Repo](https://github.com/EddieHubCommunity/LinkFree/network/members), create a branch for any issue that you are working on and commit your work.
+- [Fork the Repo](https://github.com/EddieHubCommunity/LinkFree/fork), create a branch for any issue that you are working on and commit your work.
 - Create a **[Pull Request](https://github.com/EddieHubCommunity/LinkFree/compare)** (*PR*), which will be promptly reviewed and given suggestions for improvements by the community.
 - Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes that are included in your commits.
 
 
 ## ⭐ HOW TO MAKE A PULL REQUEST:
 
-**1.** Start by making a fork the [**LinkFree**](https://github.com/EddieHubCommunity/LinkFree) repository. Click on the <a href="https://github.com/EddieHubCommunity/LinkFree/network/members"><img src="https://img.icons8.com/ios/24/32CD32/code-fork.png"></a> symbol at the top right corner.
+**1.** Start by making a fork the [**LinkFree**](https://github.com/EddieHubCommunity/LinkFree) repository. Click on the <a href="https://github.com/EddieHubCommunity/LinkFree/fork"><img src="https://seekicon.com/free-icon-download/git-fork_1.svg" height="18" width="18"></a> symbol at the top right corner.

```

Comment:
Suggest using a different color for the img

![image](https://user-images.githubusercontent.com/30869493/142629321-a02f2cc8-0842-4eb6-8b85-91498e9f39e3.png)

Reply:
I couldn't find a different version of the icon which looks great in all kinds of themes

Reply:
Hey! I have changed the icon color to green, I guess it will be visible properly in all kinds of themes